### PR TITLE
FindArray

### PR DESCRIPTION
**Pull request by Aryan-Jha29**
### Array Based

* Find Array

### CODE REVIEW

File: Binary Tree/bottomview.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* Problem Statement : Given a binary tree , print its bottom view
```
Comment (akashchouhan16) [inline Binary Tree/bottomview.cpp:1]: @Aryan-Jha29 Duplicates in this Commit.

File: Strings/DoubleEndedStrings.cpp [hunk 2]
```diff
@@ -64,7 +65,7 @@ void solve()
 }
 int32_t main()
 {
-
```
Comment (akashchouhan16) [inline Strings/DoubleEndedStrings.cpp:67]: @Aryan-Jha29  Unnecessary changes made here.

**Review by akashchouhan16 — APPROVED** [review]
Codes adhere to `CONTRIBUTING.md`, hence a valid `PR`. 👨🏻‍💻

File: Binary Tree/bottomview.cpp (hunk 1 above)
Comment (Aryan-Jha29) [reply Binary Tree/bottomview.cpp:1]: @akashchouhan16  hillloo

File: Strings/DoubleEndedStrings.cpp (hunk 2 above)
Comment (Aryan-Jha29) [reply Strings/DoubleEndedStrings.cpp:67]: @akashchouhan16 While removing merge conflict maybe some lines got changed.
